docker/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.4"
+### NETWORK     #######################################
+services:
+    postgres-network:
+        image: postgres:10
+        volumes:
+            - postgres-network:/var/lib/postgresql/data
+        expose:
+            - "5432"
+        environment:
+            POSTGRES_USER: "postgres"
+            POSTGRES_PASSWORD: "${PW_POSTGRES}"
+
+    network:
+        image: aegee/network:latest
+        volumes:
+            - network-media:/usr/app/media
+            - shared:/usr/app/shared:ro
+        depends_on:
+            - postgres-network
+        expose:
+            - "8084"
+        environment:
+            BUGSNAG_KEY: "${BUGSNAG_KEY_NETWORK}"
```

Comment:
don't forget to set it in .env.example when doing a AEGEE/MyAEGEE PR including it in general docker setup

Reply:
https://github.com/AEGEE/MyAEGEE/pull/188